waltz-ng/client/physical-flows/physical-flow-registration.js

```diff
@@ -72,6 +75,17 @@ function validate(specification, flowAttributes, targetEntity) {
 }
 
 
+function getSimilarFlows(specification, flowAttributes, targetEntity, existingFlows = []) {
```

Comment:
Want the check for similar flows to happen at the end once the user has specified all flow attributes.  I figured any earlier would interrupt the flow.

will rename to findSimilarFlows